nipype/interfaces/base.py

```diff
@@ -869,7 +858,7 @@ def _outputs_help(cls):
         """ Prints description for output parameters
         """
         helpstr = ['Outputs::', '']
-        if cls.output_spec:
+        if cls.output_spec is not None:
             outputs = cls.output_spec()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [cls.output_spec is not callable](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/1957504257/issues/source?bid=2767207&fileBranchId=2767718#l862)